standalone/src/models/shipping-method/shipping-rate/types.ts

```diff
@@ -1,16 +1,16 @@
-import { ShippingRate, ShippingRateDraft } from '@commercetools/platform-sdk';
+import { ShippingRate } from '@commercetools/platform-sdk';
 import type { TBuilder } from '@/core';
+import { TCtpShippingRate } from '@/graphql-types';
 
+/**
+ * @deprecated use `TShippingRateRest` or `TShippingRateGraphql` instead
+ */
 export type TShippingRate = ShippingRate;
-export type TShippingRateDraft = ShippingRateDraft;
```

Comment:
The draft types should be kept here for consistency with all other test-data models.